docs/comments_endpoints.md

```diff
@@ -4,8 +4,12 @@ Endpoints |	Usage |	Public Access
 --------- | ----- | -------------
 GET /answers/:id/comments |	Returns all the comments for a particular answer |	True
 POST /answers/:id/comments/ | Creates a new comment for the specified answer |	False
-PUT /comments/:id | Updates the comment with certain attributes | False
-DELETE /comments/:id | Deletes an comment | False
+PATCH /answers/:answer_id/comments/:id | Updates the specified comment of the specified answer | False
+DELETE /answers/:answer_id/comments/:id | Deletes the specified comment of the specified answer | False
+GET /questions/:id/comments |	Returns all the comments for a particular question |	True
+POST /questions/:id/comments/ | Creates a new comment for the specified question |	False
+PATCH /questions/:question_id/comment/:id | Updates the specified comment of the specified question | False
```

Comment:
I think this route ought to be `PATCH /questions/:question_id/comments/:id` not comment